docs/blog/posts/2023/single-tier-dc.md

```diff
@@ -0,0 +1,153 @@
+---
+date: 2023-06-22
+tags:
+  - evpn
+  - mc-lag
+  - datacenter
+  - multi-homing
+authors:
+  - amitk
+---
+
+# Single Tier Datacenters - Evolving Away From Multi-chassis LAG
+
+Multi-Chassis LAG (MC-LAG) was a welcome technology that helped enterprises move away from xSTP based L2 networks. It solved many of the issues inherent to xSTP networks, like underutilized links, long convergence times, and layer 2 loops. It became a common design pattern in many datacenters at the access and aggregate layers.
+
+And as with any other technology, MC-LAG started to show its limitations as the scale of the datacenters network grew. The need for a more scalable, interoperable and simpler solution led to the development of EVPN Multihoming (EVPN-MH) design. This article discusses the migration path from MC-LAG-based to EVPN-MH deployments for small, single-tier datacenters.
+
+<!-- more -->
+
+## With MC-LAG
+
+MC-LAG solutions are based on vendor proprietary control plane. Although they solved many problems with previous designs, they came with their own set of challenges:
+
+* **Split-Brain Situations:** This occurs when the two chassis lose connectivity with each other. This leads to inconsistencies, traffic loops and potential downtimes.
+
+* **Complexity:** The MC-LAG solution can be quite complex in configuration and in management. It requires a lot of careful planning and monitoring to ensure smooth operations.
+
```

Comment:
EVPN may be considered more complex in configuration aspect, while may require less maintenance.